src/algorithms/distancegradient.jl

```diff
@@ -5,15 +5,15 @@ The `sources` field is the *linear* indices of the matrix, from which the
 distance must be calculated.
 """
 struct DistanceGradient <: NeutralLandscapeMaker
-    sources::Vector{Integer}
+    sources::Vector{Int}
 end
 
 DistanceGradient() = DistanceGradient([1])
 
 function _landscape!(mat, alg::DistanceGradient)
     @assert maximum(alg.sources) <= length(mat)
     @assert minimum(alg.sources) > 0
-    coordinates = Matrix{Float64}(undef, (2, prod(size(mat))))
+    coordinates = Matrix{AbstractFloat}(undef, (2, prod(size(mat))))
```

Comment:
Here we are casting it ourself from Ints. I guess we could use Float32 here for any reasonable sizes of that matrix?